pyproject.toml

```diff
@@ -25,9 +25,9 @@ packages = [
 
 [tool.poetry.dependencies]
 python = "^3.8"
-urllib3 = "^1.26.9"
-websockets = "^10.3"
-certifi = "^2022.5.18"
+urllib3 = ">=1.26.9,<3"
```

Comment:
My knowledge on this syntax might be off, but the current expression `^1.26.9` Means we accept the versions
`>=1.26.9 <2.0.0`. From [docs](https://python-poetry.org/docs/dependency-specification/#caret-requirements).

Is this proposal to changed the allowed version to `>=1.26.9 <3.0.0`?

Reply:
Also I am inclined to merge in #439 and other dependabot PRs that might help with what you are experiencing.
I need to do more validation in the urllib upgrade since its a major version upgrade.

Reply:
yes, and I don't mind my PR vs dependabot, doesn't make a difference to me